src/components/BrowserFilter/FilterRow.react.js

```diff
@@ -92,15 +93,59 @@ let FilterRow = ({
       if (input !== null && editMode) {
         input.focus();
       }
-    }, [])    
+    }, [])
+
+    // This will add https://github.com/parse-community/parse-dashboard/pull/2463#issuecomment-1595918018
+    const buildSuggestions = (input) => {
```

Comment:
I think this whole method can be simplified to:

```
const regex = new RegExp(input.split('').join('.*?'), 'i');
return fields.filter(f => f.matches(regex));
```

That would be approach (b). Comment above method can be removed.

Reply:
I had the regex version of the code but not decided to push as it is a little bit confusing to the other contributing person. So decided to make the logic very simple.

What should I do now? Should I change it with your given code?

Reply:
Confusing in what way?

Reply:
Not confusing but a complex one.

Reply:
The regex in https://github.com/parse-community/parse-dashboard/pull/2463#discussion_r1233232407 isn't really complex?

Reply:
Yes, I'll apply it and push it. I was just mentioning that I have generated the regex with the help of AI which was complex just like https://github.com/parse-community/parse-dashboard/pull/2464#discussion_r1233236494 this one, the one in the screenshot. And that is why decided to not go with it.

Since you suggested a better one I can do that. It's just that I'm not that good with regex and thus implemented the simplest solution.

Reply:
There is slight error in ur suggestion, instead of `f => f.test(regex)` we do `f => regex.test(f)`

Reply:
I personally didn't come across an AI that generated good quality regex yet. The one I suggested should do the trick.